client/src/components/pages/database/breakdown-chart.tsx

```diff
@@ -21,8 +21,26 @@ const BreakdownChart = ({
     <table className="breakdown-chart">
       <caption></caption>
       <thead>
+        <th>License Type:</th>
+        <th>Total Available</th>
```

Comment:
```suggestion
        <tr>
          <th>License Type:</th>
          <th>Total Available</th>
        </tr>
```

Reply:
You need to surround `<th>` tags within `<tr>` routes.

We get this console error without it:
```
In HTML, <th> cannot be a child of <thead>.
This will cause a hydration error.
```

You can read about hydration [here](https://18.react.dev/reference/react-dom/client/hydrateRoot). I put it on my to read list! If you read it and learn something, I encourage you to post in the channel to share it.

With the current state of AI, the best we can do to secure our careers, is to develop a deep understanding of the tools we use.